fastlane.js

```diff
@@ -0,0 +1,52 @@
+"use strict"
+const githubUsername = process.env.HUBOT_GITHUB_USERNAME
+const githubToken = process.env.HUBOT_GITHUB_TOKEN
+const organisation = 'owncloud'
+const projectName = 'Current: QA/CI/TestAutomation'
+const fastLaneColumnName = 'Fastlane'
+const intervalInS = 3600
```

Comment:
this interval is too short IMO. I don't think we need to check this often. once every 5 minutes or 10 minutes might be probably ok.

Reply:
IMO it is 3600 seconds = 1 hour @dpakach